### PR TITLE
(feature) malware visualization changes

### PR DESCRIPTION
**Pull request by RochanaN**
Issue 9: Non-functional Buttons on "Malware Visualization - Join Us" Page. 

Description: Resolved The "Explore Projects" and "Join Us" buttons on the "Malware Visualization - Join Us" page.

### CODE REVIEW

File: home/templates/pages/malware_visualization/main.html [hunk 1]
```diff
@@ -2,107 +2,132 @@
...
+                <div class="row justify-content-center text-center">
+                    <div class="col-lg-8">
+                        <div class="mt-4 mb-5">
+                            <h2 class="fw-bolder text-dark">Join Us</h2>
+                            <p class="mb-4">
+                                Join our mission to revolutionize mobile security and combat SMS phishing
+                                attacks. Whether you're a skilled developer, cybersecurity enthusiast, or passionate
+                                advocate for online safety, there are numerous ways to get involved.
+                            </p>
+                            <a href="{% url 'smishingdetection_join_us' %}" class="btn btn-primary animate-up-2"
```
Comment (jbm1412) [inline home/templates/pages/malware_visualization/main.html:120]: Please remove Smishing Detection Join Us link for Malware Visualization Page

**Review by jbm1412 — CHANGES_REQUESTED** [review]
Page looks good but fix the change

**Comment by RochanaN** [comment]
@jbm1412 I have removed smishing detection link

**Comment by s-bonnor** [comment]
This isnt mergeable right now as it seems to be trying to delete a lot of stuff that was added this trimester

**Comment by RochanaN** [comment]
> This isnt mergeable right now as it seems to be trying to delete a lot of stuff that was added this trimester

can you tell me the files that i have changed?

**Comment by s-bonnor** [comment]
if you look at the files changed tab you can see that about 20 files are deleted and many more have lines in them deleted

**Comment by RochanaN** [comment]
i have done 2 changes so far. have you merged the first changes to the main code? (malware visualization ...)

**Pull request closed without merging**